src/openapi/streaming/connection/transport/websocket-transport.js

```diff
@@ -170,6 +170,14 @@ function handleSocketClose(event) {
             code: event.code,
         });
 
+        // 1006 doesn't tell us its a 401 unauthorized, just that it might be
+        // the websocket spec disallows reading status codes for security reasons
+        // (so websocket can't be used to probe http endpoints)
+        // So, before we reconnect, lets re-authorize to be sure
+        if (event.code === 1006) {
+            this.getAuthorizePromise(this.contextId, true);

```

Comment:
the idea being that because authorize uses the auth transport, a 401 will be handled through the auth transport

Reply:
Should this comment also be part of the code?

Reply:
yes you're right, added.